apps/site/src/lib/components/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,37 @@
+.navMenuListWrapper {
+	// clip top only when mobile
+	@media (max-width: 768px) {
+		clip-path: inset(0px 0px -100px -100px);
+	}
+}
+
+.navMenuLink {
+	$underline-size: 2px;
+
+	font-size: 24px;
+	text-shadow: 0px 0px 20px #fffce2;
+
+	// needed for underline effect to work
+	padding-bottom: 0.5rem;
+
+	filter: brightness(1.05);
+
+	background-image: linear-gradient(white, white);
+	background-repeat: no-repeat;
+	background-position-y: 34px;
+	background-size: 0% $underline-size;
+	transition:
+		filter 0.1s ease,
+		text-shadow 0.1s ease,
+		background-size 0.25s ease;
+
+	&:hover,
+	&:focus,
+	&:active {
+		text-shadow:
+			0px 0px 20px #ffa700,
+			0px 0px 20px #ffda7b;
+		filter: brightness(3.05);
```

Comment:
Issue: The filter is still causing some flashing glitches on webkit. Given this is being applied to just text and it's shadow, could we replace the filter effect with a brighter text color and/or text-shadow color?